website/docs/docs/core/connect-data-platform/cratedb-setup.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "CrateDB setup"
+description: "Read this guide to learn about the CrateDB warehouse setup in dbt."
+id: "cratedb-setup"
+meta:
+  maintained_by: Crate.io, Inc.
+  authors: 'CrateDB maintainers'
+  github_repo: 'crate/dbt-cratedb2'
+  pypi_package: 'dbt-cratedb2'
+  min_core_version: 'v1.0.0'
+  cloud_support: Not Supported
+  min_supported_version: 'n/a'
+  slack_channel_name: 'Community Forum'
+  slack_channel_link: 'https://community.cratedb.com/'
+  platform_name: 'CrateDB'
+  config_page: '/reference/resource-configs/postgres-configs'
+---
+
+import SetUpPages from '/snippets/_setup-pages-intro.md';
+
+<SetUpPages meta={frontMatter.meta}/>
+
+
+[CrateDB] is compatible with PostgreSQL, so its dbt adapter strongly depends on
+dbt-postgres, documented at [Postgres profile setup].
+
+CrateDB targets are configured exactly the same way, with just a few things to
+consider which are special to CrateDB. Relevant details are outlined at [using
+dbt with CrateDB], which also includes up-to-date information.
+
+
+## Profile configuration
+
+CrateDB targets should be set up using a configuration like this minimal sample
+of settings in your [`profiles.yml`] file.
+
+<File name='~/.dbt/profiles.yml'>
+
+```yaml
+cratedb_analytics:
+  target: dev
+  outputs:
+    dev:
+      type: cratedb
+      host: [clustername].aks1.westeurope.azure.cratedb.net
+      port: 5432
+      user: [username]
+      pass: [password]
+      dbname: crate     # CrateDB's only catalog is `crate`.
+      schema: doc       # Define schema. `doc` is the default.
+      search_path: doc  # Use the same value like `schema` by default.
```

Comment:
What is the difference between schema and search path (or maybe the better question is how are they related?)

Reply:
We have a bit of information about `search_path` on the documentation page about [CrateDB session settings](https://cratedb.com/docs/crate/reference/en/latest/config/session.html). In general, it is the same like the [PostgreSQL setup](https://docs.getdbt.com/docs/core/connect-data-platform/postgres-setup) page says:

> [search_path](https://docs.getdbt.com/docs/core/connect-data-platform/postgres-setup#search_path): [optional, override the default postgres `search_path`]

Maybe we should align with the PostgreSQL page, but explicitly expand into stating the exact differences to PostgreSQL:

> The `search_path` config controls the Postgres "search path" that dbt configures when opening new connections to the database. By default, the Postgres search path is ~~"$user, public"~~ "pg_catalog, doc", meaning that unqualified table names will be searched for in the `doc` schema, ~~or a schema with the same name as the logged-in user.~~ **Note:** Setting the `search_path` to a custom value is not necessary or recommended for typical usage of dbt.

In this case, it might be advisable to remove this setting completely from the minimal set of settings we are presenting on the upcoming [CrateDB setup page](https://docs-getdbt-com-git-fork-crate-workbench-cratedb-dbt-labs.vercel.app/docs/core/connect-data-platform/cratedb-setup), and only describe it on the [CrateDB and dbt documentation page](https://cratedb.com/docs/guide/integrate/dbt/) instead. wdyt? /cc @kneth

Can you please also review this, @seut and @hlcianfagna, to confirm that this also holds true for CrateDB:

> **Note:** Setting the `search_path` to a custom value is not necessary or recommended for typical usage of dbt.

Reply:
Hi again. I've just updated the patch, removing the option about `search_path`. It also works without this setting. We decided to keep the basic configuration section minimal, and the CrateDB setup page is referring to the PostgreSQL setup page anyway.

Please let us know if you think differently, @kneth, @hlcianfagna, or @seut, possibly also in retrospective.

Reply:
It is an advanced option, I agree it may be better omitted in the minimal example as it is optional and may lead to confusion.

Reply:
Thank you. Resolving this.